test/transactsql.test.ts

```diff
@@ -118,27 +118,37 @@ describe('TransactSqlFormatter', () => {
     `);
   });
 
-  it('formats SET NOCOUNT ON; GO CREATE OR ALTER TABLE', () => {
-    const result = format('SET NOCOUNT ON; GO CREATE OR ALTER TABLE t');
+  it('formats SET NOCOUNT ON; GO CREATE OR ALTER PROCEDURE', () => {
+    const result = format('SET NOCOUNT ON; GO CREATE OR ALTER PROCEDURE p');
     expect(result).toBe(dedent`
     SET
     NOCOUNT ON;
 
     GO
-    CREATE OR ALTER TABLE
-      t
+    CREATE OR ALTER PROCEDURE
+      p
     `);
   });
 
-  it('formats SET NOCOUNT ON; GO CREATE OR ALTER PROCEDURE', () => {
-    const result = format('SET NOCOUNT ON; GO CREATE OR ALTER PROCEDURE p');
+  it('formats SELECT ... INTO clause', () => {
+    const result = format('SELECT col INTO #temp FROM tbl');
     expect(result).toBe(dedent`
-    SET
-    NOCOUNT ON;
+    SELECT
+      col
+    INTO
+      #temp
+    FROM
+      tbl
+    `);
+  });
 
-    GO
-    CREATE OR ALTER PROCEDURE
-      p
+  it('formats INSERT INTO', () => {
+    const result = format("INSERT INTO tbl(firstName, lastName) VALUES('George', 'Orwell')");
+    expect(result).toBe(dedent`
+    INSERT INTO
+      tbl (firstName, lastName)
+    VALUES
+      ('George', 'Orwell')
     `);
   });
```

Comment:
This test is not needed. It's already covered by the `supportsInsertInto()` call to a shared test.

Reply:
Confirming redundant test has been removed